### ISSUE TITLE
Expected accuracy for Sentence Classifier - accuracy currently 25%?

### ISSUE THREAD

**Issue by katelyn98** [salaniz/pytorch-gve-lrcn#20]
I am training the sentence classifier on CUB to ultimately train the GVE model, but my model converges around 10 epochs at about 25% accuracy. Has anyone reached higher accuracy for their sentence classifier? I am using 1 GPU and following the default scripts that were given. Thanks!

**Comment by katelyn98** [comment]
I guess the accuracy doesn't matter for the sentence classifier? I was able to successfully train the model and get reasonable visual explanations out of it. Thanks!
